dask_expr/io/_delayed.py

```diff
@@ -23,6 +23,7 @@ class _DelayedExpr(Expr):
     _parameters = ["obj"]
 
     def __init__(self, obj):
+        self._simplified = {}
```

Comment:
This confuses me, can you elaborate?

Reply:
Oops - this was leftover from an earlier experiment. I'll remove it. Thanks!